docs/running-on-mesos.md

```diff
@@ -153,6 +153,8 @@ can find the results of the driver from the Mesos Web UI.
 To use cluster mode, you must start the `MesosClusterDispatcher` in your cluster via the `sbin/start-mesos-dispatcher.sh` script,
 passing in the Mesos master URL (e.g: mesos://host:5050). This starts the `MesosClusterDispatcher` as a daemon running on the host.
 
+By setting the mesos proxy config property (requires mesos version >= 1.4), eg. `--conf spark.mesos.proxy.base_url=http://localhost:5050` when launching the dispacther, the mesos sandbox uri for each driver is added on the mesos dispatcher ui.
```

Comment:
mesos -> Mesos
eg. -> e.g. (but really, should be removed? it's not an example)
uri -> URI
ui -> UI
"added on" -> "added to" ?